vector/src/main/java/im/vector/app/features/login/LoginViewModel.kt

```diff
@@ -63,20 +65,30 @@ class LoginViewModel @AssistedInject constructor(
         private val activeSessionHolder: ActiveSessionHolder,
         private val homeServerConnectionConfigFactory: HomeServerConnectionConfigFactory,
         private val reAuthHelper: ReAuthHelper,
-        private val stringProvider: StringProvider
+        private val stringProvider: StringProvider,
+        private val homeServerHistoryService: HomeServerHistoryService
 ) : VectorViewModel<LoginViewState, LoginAction, LoginViewEvents>(initialState) {
 
     @AssistedInject.Factory
     interface Factory {
         fun create(initialState: LoginViewState): LoginViewModel
     }
 
+    init {
+        if (BuildConfig.DEBUG) {
+            homeServerHistoryService.addHomeServerToHistory("http://10.0.2.2:8080")
```

Comment:
I would not modify the DB here, but just add this debug URL to the list returned by `getKnownServersUrls()`

Reply:
Done